lib/features/composer/presentation/composer_controller.dart

```diff
@@ -1686,7 +1720,9 @@ class ComposerController extends BaseController with DragDropFileMixin implement
     }
 
     if (newIdentity.signatureAsString.isNotEmpty == true) {
-      await _applySignature(newIdentity.signatureAsString.asSignatureHtml());
+      await Future.delayed(const Duration(milliseconds: 300), () {
+        _applySignature(newIdentity.signatureAsString.asSignatureHtml());
+      });
```

Comment:
Why do we need delay time for it?

Reply:
After calling `emailContentsViewState.value = Right(success);`, the UI need some time to load. If I call `_applySignature` right away, nothing will happen, and the signature button fails to restored

Reply:
So is `300 ms` enough to do that. Can we write a unit test for it?

Reply:
No this cannot be unit tested